readthedocs/projects/tasks.py

```diff
@@ -943,18 +944,23 @@ def _manage_imported_files(version, path, commit):
     changed_files = set()
     for root, __, filenames in os.walk(path):
         for filename in filenames:
+            if fnmatch.fnmatch(filename, '*.html'):
+                model_class = HTMLFile
+            else:
+                model_class = ImportedFile
```

Comment:
I don't believe this is needed, since it's all the same model in the database.

Reply:
The problem is with actually signal manager. I have opened https://github.com/sabricot/django-elasticsearch-dsl/issues/111 about this.

Untill it has been fixed, we need to have a proxy model for the purpose, I believe.